oblib/tests/test_validator.py

```diff
@@ -31,13 +31,32 @@ def test_validate_concept_value(self):
 
         concept.type_name = "xbrli:booleanItemType"
         self.assertEqual(1, len(validator.validate_concept_value(concept, "Arf")))
+        self.assertEqual(1, len(validator.validate_concept_value(concept, "0100")))
```

Comment:
We're changing the signature of each validator function to return a tuple `value, errors`, so all test statements need to change.  I'd suggest this pattern:

result = validator.validate_concept_value(concept, test_value)
self.assertEqual(expected, result[0])
self.assertTrue(result[1])  or self.assertFalse(result[1]) if the test is expected to fail